### ISSUE TITLE
Kubedmp does not show the external IPs for LoadBalancer type services

### ISSUE THREAD

**Issue by eleblebici** [shundezhang/kubedmp#2]
```
$ kubedmp -v
Version:	 0.8.1
build.Time:	 2024-03-19 11:50:50.679476541 +0000 UTC
build.User:	 shundezhang
```

```
$ kubedmp get svc traefik -n cos
NAMESPACE	NAME	TYPE		CLUSTER-IP	EXTERNAL-IP	PORT(S)		AGE
cos		traefik	LoadBalancer	10.152.183.157	<none>		80/TCP,443/TCP	73d
```

`cos                              service/traefik                                     LoadBalancer   10.152.183.157   10.4.26.228   80:32246/TCP,443:30302/TCP    72d     app.kubernetes.io/name=traefik`

I took these samples from an inspection report of microk8s. Though we have the externap IP in `kubectl get all` output, kubedmp does not show the external IP for the LoadBalancer service traefik.

**Comment by shundezhang** [comment]
@eleblebici Thank you for reporting the issue. Just realised I forgot to implement the code to get a value for external ip. Working on it now...

**Comment by shundezhang** [comment]
I've released 0.8.2 with a fix. Please try and let me know if there is any problem.

**Comment by eleblebici** [comment]
Thank you @shundezhang It seems I'm using the version 0.8.3 now and it seems it is fixed.

```
$ kubedmp -v
Version:	 0.8.3
build.Time:	 2024-05-03 06:16:12.690928217 +0000 UTC
build.User:	 shundezhang
birru:~/Documents/Canonical/Cases/384268/inspection-report/k8s$ kubedmp get svc -A | egrep -v none
NAMESPACE		NAME					TYPE		CLUSTER-IP	EXTERNAL-IP				PORT(S)					AGE
kubeflow		istio-ingressgateway-workload		LoadBalancer	10.152.183.57	10.117.47.43				80/TCP,443/TCP				13d
```
